modules/administration/pages/custom-channels.adoc

```diff
@@ -35,23 +37,29 @@ Do not create child channels containing packages that are not compatible with th
 
 
 .Procedure: Creating a Custom Channel
-
 . In the {productname} {webui}, navigate to menu:Software[Manage > Channels], and click btn:[Create Channel].
 . On the [guimenu]``Create Software Channel`` page, give your channel a name (for example, [systemitem]``My Tools SLES 15 SP1 x86_64``) and a label (for example, [systemitem]``my-tools-sles15sp1-x86_64``).
-Labels must not contain space characters or capital letters.
+Labels must not contain spaces or capital letters.
```

Comment:
We usually call those uppercase letters. At least in German, "capitals" are a way to layout letters.

Reply:
Yeah, I was wondering about that.